tests/Twilio/Unit/Rest/ClientTest.php

```diff
@@ -241,4 +241,20 @@ public function testAuthorizationHeaderRemoval(): void {
         $this->assertStringNotContainsString('Authorization-header', stream_get_contents($capturedLogging));
     }
 
+    public function testDefaultUserAgent(): void {
+        $client = new Client('username', 'password');
+        $client->request('GET', 'https://api.twilio.com');
+        $userAgent = $client->getHttpClient()->{'lastRequest'}[CURLOPT_HTTPHEADER][0];
+        $this->assertRegExp('/^User-Agent: twilio-php\/[0-9.]*\s[(][\w]*\s[\w]*[)]\sPHP\/[0-9.]*/',$userAgent);
```

Comment:
Prefer to use escape character instead of brackets for some of the characters. E.g., instead of `[(]` use `\(`

Should also match on at least one (`+`) instead of at least one (`*`) in several places. This ensures it was seen once.

Should also end the sequence with a `$` to match the end of the string.